api/types.proto

```diff
@@ -696,3 +696,9 @@ message ManagerStatus {
 	// Reachability specifies whether this node is reachable.
 	RaftMemberStatus.Reachability reachability = 4;
 }
+
+message Device {
+	string source = 1;
+	string target = 2;
+	string permissions = 3;
```

Comment:
I don't have a solid suggestion. Research needs to be done to clarify the minimum set of fields necessary to express this correctly without making the fields too host specific.

engine-api was designed before libcontainer and I'd like to understand the separation.